### PR TITLE
dr-ausosync: make report interval configurable

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What is changed and how it works?

Issue Number: ref https://github.com/tikv/pd/issues/4686

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

`kubectl edit tc -n pd -o yaml`
image
image

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: components/raftstore/src/store/worker/pd.rs [hunk 1]
```diff
@@ -457,35 +465,34 @@ where
...
                             store_id,
                             &scheduler,
                         );
                     }
-                    // modules timer_cnt with the least common multiple of intervals to avoid overflow
-                    timer_cnt = (timer_cnt + 1)
-                        % (load_base_split_check_interval
-                            * collect_store_infos_interval
-                            * report_min_resolved_ts_interval);
+                    timer_cnt += 1;
```
Comment (AndreMouche) [inline components/raftstore/src/store/worker/pd.rs:495]: would `timer_cnt` overflow?

**Review by AndreMouche — COMMENTED** [review]
Rest LGTM

File: components/raftstore/src/store/worker/pd.rs (hunk 1 above)
Comment (lhy1024) [reply components/raftstore/src/store/worker/pd.rs:495]: Currently running at the interval of one second, which I think uint64 is sufficient.

File: components/raftstore/src/store/config.rs [hunk 2]
```diff
@@ -355,6 +358,7 @@ impl Default for Config {
             region_split_size: ReadableSize(0),
             clean_stale_peer_delay: ReadableDuration::minutes(0),
             inspect_interval: ReadableDuration::millis(500),
+            report_min_resolved_ts_interval: ReadableDuration::millis(0),
```
Comment (nolouch) [inline components/raftstore/src/store/config.rs:361]: Off by default?

Comment (lhy1024) [reply components/raftstore/src/store/config.rs:361]: yes, advice by @disksing

**Review by nolouch — APPROVED** [review]
lgtm

**Comment by nolouch** [comment]
ci failed @lhy1024 
image

**Comment by lhy1024** [comment]
manual test
image
image

**Pull request merged**